doc/man3/EVP_PKEY_fromid.pod

```diff
@@ -0,0 +1,59 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_param_fromid, EVP_PKEY_key_fromid
+- functions to create domain parameters and keys from an identity
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ int EVP_PKEY_param_fromid(EVP_PKEY_CTX *ctx, EVP_PKEY **ppkey,
+                           void *id, size_t idlen);
+ int EVP_PKEY_key_fromid(EVP_PKEY_CTX *ctx, EVP_PKEY **ppkey,
+                         void *id, size_t idlen);
+
+=head1 DESCRIPTION
+
+EVP_PKEY_param_fromid() creates domain parameters, given an identity
+I<id> of length I<idlen>.  The result is written to I<*ppkey>.
+
+EVP_PKEY_key_fromid() creates domain parameters, given an identity
+I<id> of length I<idlen>.  The result is written to I<*ppkey>.
+
+In both cases, the identity is a byte string.  Exactly what that byte
+string contains is undefined here.  This depends on the provider and
+what it handles.
+
```

Comment:
This page would probably benefit from an example of some form (not necessarily a worked example of using the API; a concrete statement of what parameters are created from a particular identity string might suffice).

Reply:
I'm not sure I understand.  Doesn''t "The result is written to I\<*ppkey\>." answer that question?

Reply:
I was thinking in the general vein of "such as DSA parameters p, q, and g L<openssl-dsaparam(1)>"

Reply:
So basically, what we actually need is a general description of a `EVP_PKEY`, and link to that.

Reply:
That description could match something quite useful, yes.  (It could also match something else, so ... I'll try to look at a PR if one appears.)